### ISSUE TITLE
Bump core so that it works with Job DSL

### ISSUE THREAD

**Issue by MadsNielsen** [Praqma/pretested-integration-plugin#103]
We should probably update to 2.19.4. That is what is the required core.

See issue #102 we get this kind of error with HPI run

**Comment by MadsNielsen** [comment]
First attempt:

```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for com.google.code.findbugs:annotations:3.0.0 paths to dependency are:
+-org.jenkins-ci.plugins:pretested-integration:3.0.0-SNAPSHOT
  +-com.google.code.findbugs:annotations:3.0.0
and
+-org.jenkins-ci.plugins:pretested-integration:3.0.0-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.19.4
    +-org.jenkins-ci.main:remoting:2.62.3
      +-com.google.code.findbugs:annotations:3.0.0 (managed) <-- com.google.code.findbugs:annotations:3.0.1
```

**Comment by MadsNielsen** [comment]
Second try. Manually adding the dependency: 

Added 
```
        <!-- Temparily added this because of issue GHI 103 -->
        <dependency>
            <groupId>com.google.code.findbugs</groupId>
            <artifactId>annotations</artifactId>
            <version>3.0.1</version>
        </dependency>
```

Got this startup error:
```
SEVERE: Failed Loading plugin Jenkins Git plugin v3.5.1 (git)
java.io.IOException: Jenkins Git plugin v3.5.1 failed to load.
 - Jenkins Parameterized Trigger plugin v2.25 is older than required. To fix, install v2.33 or later.
 - Token Macro Plugin v1.10 is older than required. To fix, install v1.12.1 or later.
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:621)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:516)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$7.runTask(Jenkins.java:1085)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by buep** [comment]
Related to #64 - needed to write tests. See my comment here: https://github.com/Praqma/pretested-integration-plugin/issues/102#issuecomment-357271168
